CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Add pace and duration icons.
```

Comment:
in your commit message:
- Add a `COURSE_RUN_SYNC_PROTECTED_FIELDS` **setting**
- s/we are allow/we are allowing

It's no more a dict but a list